src/gtk/widgets/info_bar.rs

```diff
@@ -21,7 +21,7 @@ use glib::translate::ToGlibPtr;
 use gtk::MessageType;
 use gtk::cast::GTK_INFOBAR;
 use gtk::{self, ffi};
-use glib::{to_bool, to_gboolean};
+use glib::to_gboolean;
```

Comment:
Hehe, you too have fallen for removing imports that aren't needed with GTK_3_6 ;)
I'm thinking, maybe we should gather the conversion stuff up into a crate-private prelude that gets included everywhere by glob (there's an RFC to make it automatic).